ci/ci-test-tasks/test-and-verify-v2.js

```diff
@@ -112,7 +112,11 @@ function runTestPipeline(pipeline, config = '') {
         CANARY_TEST_CONFIG: {
           "isSecret": false,
           "value": config 
-        }
+        },
+        CANARY_TEST_NODE_VERSION: {
+          "isSecret": false,
+          "value": CANARY_TEST_NODE_VERSION
```

Comment:
will this work in case we don't have any value?

Reply:
yes, the extension will fall back to node 16 if this value is not specified.